docs/Coding-Guidelines.md

```diff
@@ -57,3 +57,66 @@ ABI.
 GitHub diff is line oriented. Keeping lines within the preferred limit makes changes easier to
 review. Use a tool like Prettier to bulk-reformat files, or configure your editor's "rulers." If new
 languages become common, we will describe the coding guidelines for such languages here.
+
+## Git Checkin-to-main Policy
+
+The main branch needs to be kept 'healthy' and ready to ship near daily (daily, or within a day or
+two). An unhealthy main branch negatively impacts day-to-day development and jeopardizes release
+schedules. We have policies to ensure a high quality bar in main. The details differ depending if a
+code change is Experimental vs Preview or Stable.
+
+See [Final Decision: Release channels and cadences #637](https://github.com/microsoft/WindowsAppSDK/discussions/637)
+for the expectations of the various release channels.
+
+### Policy: Preview and Stable (aka non-Experimental)
+
+The following requirements must be met to checkin non-Experimental changes to main:
+
+1. API Review approval
```

Comment:
Traditionally, we don't require API review to be completed before something RIs to rsmain. We just require API review before Windows ships (i.e., the GA build of Reunion ships). Are we changing policy here?

Reply:
That's fine as long as it's [experimental]. We shouldn't commit an API (remove the tag) until we've reviewed it.

Reply:
For Windows in-box, our guidance for multiple releases has been that APIs should be under experimental/feature configuration until they have been fully reviewed, so they show up in the prerelease SDK as "experimental" indicating that they might change. This is the same guidance - "mark your unfinished unreviewed APIs as experimental" ... the implication of this section is that if you merge to main you EITHER:

* Have finished your API review and it's in the final state
* Have not finished your API review and your API is marked experimental

(Note that some teams in-box forget to use the tagging, which occasionally leads to sadness.)